packages/odyssey-react-mui/src/labs/DataComponents/RowActions.tsx

```diff
@@ -0,0 +1,122 @@
+/*!
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Fragment, ReactElement, memo, useCallback } from "react";
+import { MRT_Row, MRT_RowData } from "material-react-table";
+import { useTranslation } from "react-i18next";
+
+import {
+  ArrowBottomIcon,
+  ArrowDownIcon,
+  ArrowTopIcon,
+  ArrowUpIcon,
+} from "../../icons.generated";
+import { Button } from "../../Button";
+import { MenuItem } from "../../MenuItem";
+import { DataTableProps } from "./DataTable";
+import { MenuButtonProps } from "../../MenuButton";
+
+export type RowActionsProps = {
+  isRowReorderingDisabled?: boolean;
+  row: MRT_Row<MRT_RowData> | MRT_RowData;
+  rowActionButtons?: (
+    row: MRT_RowData,
+  ) => ReactElement<typeof Button | typeof Fragment>;
```

Comment:
Does this work the way we expect?

Reply:
Nope, it doesn't — turns out it needs to be `ReactElement<typeof Button> | ReactElement<typeof Fragment>`.